lessons/codap-exploring/langs/en-us/pages/codap-exploration.adoc

```diff
@@ -1,50 +1,100 @@
 
-@n Minimize and maximize the table and the graph. How do you do that?
+@n How many animals have names that begin with S? @fitb{5em}{@ifsoln{6}}
 
-@fitb{}{@ifsoln{ To minimize, I select the `-` in the upper right-hand corner of the graph or table. To maximize, I reselect the `-`.}}
+@n Delete a column of the table.
 
-@fitb{}{}
+@n Close the table. __Can you get it back?__ (Hint: A drop-down menu appears when you click on "Tables" in the upper left.)
 
-@vspace{2ex}
+@n Create a new attribute. Is the column populated or empty? @fitb{15em}{@ifsoln{Empty.}}
 
-@n Drag an attribute (like `weight`, `name`, or `sex`) to a graph's axis - or use the drop-down menu that appears when you click on an axis. What happens?
+@n Name your new attribute. What did you choose? @fitb{15em}{}
 
-@fitb{}{@ifsoln{ _The randomly positioned points move to the confirguration that matches the data on the animals table._}}
+@n Do Undo (`Ctrl-Z` on PC, `Cmd-Z` on Mac) and Redo (`Ctrl-Y` on PC, `Cmd-Opt-Z` on Mac) work in CODAP? @fitb{}{@ifsoln{yess}}
 
-@fitb{}{}
+== Graphs in CODAP
 
-@vspace{2ex}
+@n See if you can make a graph. What did you select in order to do so? @fitb{}{@ifsoln{I clicked on the "Graph" icon in the upper left-hand corner of my screen}}
 
-@n Drag a second attribute (like `weight`, `name`, or `sex`) to the graph's other axis - or use the drop-down menu that appears when you click on an axis. What happens?
+_Note: When you first make a graph, the points are randomly positioned!_
 
-@fitb{}{@ifsoln{ The points move again, to the confirguration indicated by the two attributes on the animals table.}}
+@n Click on a dot. What happens? @fitb{}{@ifsoln{I see information about the point.}}
 
-@fitb{}{}
+@n Can you figure out a way to make _different_ information appear when you click on a dot? (Hint: You may need to move a column!)
 
-@vspace{2ex}
+@n Drag an attribute (like `Weight`, `Name`, or `Sex`) to one of the graph's axes - or use the drop-down menu that appears when you click on an axis.
 
-@n Select the grey plus sign (+) in the upper right-hand corner of the Animals Table. Doing so will enable you to create a new attribute. Name your new attribute `test`.  Is the column populated or empty?
+@n Can you make the graph show _two_ attributes?
 
-@fitb{}{@ifsoln{The column is empty.}}
+@n Double click on the background of your graph. Try doing that (zooming in) a few times.
 
-@fitb{}{}
+@n Click on the "Rescale" icon (it looks like four arrows pointing in four different directions) to zoom back out and display all data again.
 
+@n Once a graph shows two attributes, can you change it back to a graph with one attribute?
+
+@n Click and drag any attribute name from the top of any column in the dataset to the center of the graph. When the graph region turns yellow, release the mouse. What happened? @fitb{}{@ifsoln{A color-coded legend was created!}}
+
+== Matching
+
```

Comment:
This is a nice addition. I wouldn't be able to do it without having actually done the explorations (which is a good thing!) and having done the exploration it is clear to me.